### PR TITLE
<PlayerSelectionDialog/> in Craft tutorials, part 2

### PR DESCRIPTION
**Pull request by maddiedierker**
Follow-up to #41059, which introduced `<PlayerSelectionDialog/>` and refactored Minecraft "agent" to use it (instead of playerSelection.html.ejs).

This PR refactors the remaining Minecraft tutorials (simple, designer, and aquatic) to use `<PlayerSelectionDialog/>` as well. This is a pure refactor so there should be no user-facing change.

There were/are basically two versions of this dialog:

**Designer:**
Screen Shot 2021-06-22 at 8 57 34 AM

**All other Minecraft tutorials:**
Screen Shot 2021-06-22 at 8 57 25 AM

## Links

- [STAR-1553](https://codedotorg.atlassian.net/browse/STAR-1553)
- Example levels that use this dialog:
  - agent: /s/hero/lessons/1/levels/1
  - designer: /s/minecraft/lessons/1/levels/4
  - simple: /s/mc/lessons/1/levels/1
  - aquatic: /s/aquatic/lessons/1/levels/1

## Testing story

Relies on existing UI/eyes tests on Minecraft dialogs. Adds unit tests to new helper method.

### CODE REVIEW

**Comment by maddiedierker** [comment]
@code-dot-org/starlabs PTAL

File: apps/test/unit/craft/utilsTest.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+    beforeEach(() => {
+      sinon.stub(craftRedux, 'closePlayerSelectionDialog');
+    });
+
+    afterEach(() => {
+      sinon.restore();
+    });
+
+    it('closes player selection dialog after opening', () => {
```
Comment (ajpal) [inline apps/test/unit/craft/utilsTest.js]: nit: closes the dialog after selecting a player? this test name sort of makes it sound like it opens then closes immediately

**Review by ajpal — APPROVED** [review]
this is great cleanup!

**Comment by maddiedierker** [comment]
merging this into #41059 so that if i need to revert, it will be contained to 1 PR

**Pull request merged**